pkg/controller/controller_instance_test.go

```diff
@@ -1136,6 +1226,137 @@ func TestPollServiceInstanceSuccessDeprovisioningWithOperationWithFinalizer(t *t
 	assertNumEvents(t, events, 1)
 }
 
+func TestSetInstanceCondition(t *testing.T) {
+	instanceWithCondition := func(condition *v1alpha1.InstanceCondition) *v1alpha1.Instance {
+		instance := getTestInstance()
+		instance.Status = v1alpha1.InstanceStatus{
+			Conditions: []v1alpha1.InstanceCondition{*condition},
+		}
+
+		return instance
+	}
+
+	newTs := metav1.Now()
+
```

Comment:
condition is shortcut for creating a fresh condition with a default transition time of five minutes in the past.